src/components/IntlTelInput.js

```diff
@@ -522,13 +522,19 @@ class IntlTelInput extends Component {
     }
   }
 
-  loadAutoCountry = () => {
-    // check for localStorage
-    const lsAutoCountry =
-      window.localStorage !== undefined
+  loadCountryFromLocalStorage = () => {
+    try {
+      return window.localStorage !== undefined
```

Comment:
Since Babel is set up, it's probably worth it to add in the optional chaining plugin and collapse this into something like

```
loadCountryFromLocalStorage = () => window.localStorage?.getItem('itiAutoCountry')
```

It would make it succinct enough to just keep it as one function (`loadAutoCountry`) while keeping it safer re:localStorage not being available.

Reply:
For reference: https://babeljs.io/docs/en/next/babel-plugin-syntax-optional-chaining.html

Reply:
This a good suggestion, but in this case, `localStrage` won't be undefined but rather throw if you try to access it from a prohibited context. Since we'd end up wrapping a separate context with a `try-catch` statement to prevent that, the reason why we're catching may not be as clear.

Reply:
That makes sense!